build/build-race-binaries.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Build cockroach binary with race detection enabled.
+
+set -euo pipefail
+
+source $(dirname $0)/build-common.sh
+
+# This is mildly tricky: This script runs itself recursively. The
+# first time it is run it does not take the if-branch below and
+# executes on the host computer. It uses the builder.sh script to run
+# itself inside of docker passing "docker" as the argument causing the
+# commands in the if-branch to be executed within the docker
+# container.
+if [ "${1-}" = "docker" ]; then
+    time make STATIC=1 build GOFLAGS="-race"
+    check_static cockroach
+    strip -S cockroach
```

Comment:
unrelated, but looks like we ought to be using `ldflags=-s` instead:

https://stackoverflow.com/questions/29599209/how-to-build-a-release-version-binary-in-go

Wonder whether that does the same thing.